distribution/lib/Standard/Base/0.0.0-dev/src/System/File.enso

```diff
@@ -1090,3 +1056,39 @@ Text.write path encoding=Encoding.utf_8 on_existing_file=Existing_File_Behavior.
     file = new path
     on_existing_file.write file stream->
         stream.write_bytes bytes
+
+## Writes (or appends) the Vector of bytes into the specified file. The behavior
+   specified in the `existing_file` parameter will be used if the file exists.
+
+   Arguments:
+   - path: The path to the target file.
+   - on_existing_file: Specifies how to proceed if the file already exists.
+
+   If the Vector contains any item which is not a `Byte`, an
+   `Illegal_Argument_Error` will be raised.
+   If the path to the parent location cannot be found or the filename is
+   invalid, a `File_Not_Found` is raised.
+   If another error occurs, such as access denied, an `Io_Error` is raised.
+   Otherwise, the file is created with the encoded text written to it.
+
+   > Example
+     Write the bytes of the text "hello" to a file.
+
+         import Standard.Examples
+
+         example_write_bytes = "hello".utf_8.write_bytes Examples.scratch_file
+   > Example
+     Append the bytes of the text "hello" to a file.
+
+         import Standard.Examples
+
+         example_append_bytes = "hello".utf_8.write_bytes Examples.scratch_file Existing_File_Behavior.Append
```

Comment:
This example essentially works as `Text.write`. I'd suggest to show an example where we write non-text bytes, like `[1, 2, 0, 44, 255].write_bytes ...`.

Also from the documentation I don't know how the Enso bytes are supposed to be stored. We don't have a byte type but just Integer. If I want to have `0xFF` do I go for `255` or `-127`? Or do both work? I think this function is a right place to mention this. IIRC it would be `-127`; I remember we had a discussion if we could run a preprocess step on the Enso array which would check if the bytes are within bounds and translate the `255`s into `-127`s etc. Do we still want to do that?